library/Requests/Transport/cURL.php

```diff
@@ -135,7 +135,11 @@ public function request($url, $headers = array(), $data = array(), $options = ar
 		$options['hooks']->dispatch('curl.before_send', array(&$this->handle));
 
 		if ($options['filename'] !== false) {
-			$this->stream_handle = fopen($options['filename'], 'wb');
+			@$this->stream_handle = fopen($options['filename'], 'wb');
```

Comment:
The error silencing should be as limited as possible, so we want ot have it directly in front of the problematic `fopen()` call.
```suggestion
			$this->stream_handle = @fopen($options['filename'], 'wb');
```